src/Views/Movies/Create.cshtml

```diff
@@ -4,7 +4,7 @@
     ViewData["Title"] = "Create Movie";
 }
 
-<form asp-controller="Movies" asp-action="Create" method="post" class="form-horizontal">
+<form asp-controller="MovieViewModel" asp-action="Create" method="POST" class="form-horizontal">
```

Comment:
i think that 'asp-controller' must be "Movies". MovieViewModel isn't a controller